tests/bluetooth/bsim_bt/bsim_test_mesh/_mesh_test.sh

```diff
@@ -9,7 +9,7 @@ function Execute(){
  compile it?)\e[39m"
     exit 1
   fi
-  timeout 20 $@ & process_ids="$process_ids $!"
+  timeout 60 $@ & process_ids="$process_ids $!"
```

Comment:
Just increase it more. This is just a safety break that should be much bigger than the time it normally takes.

Reply:
Increased to 5 minutes